content/en-US/api/fontVariantCaps.md

```diff
@@ -0,0 +1,102 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontVariantCaps'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontVariantCaps
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontVariantCaps
+---
+
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontVariantCaps](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontVariantCaps).**
+
+## Description
+
+The `fontVariantCaps` property of `CanvasRenderingContext2D` allows developers
+to control the use of alternate glyphs for capital letters. If multiple size 
+of capital letter glyphs are available for the chosen font, this 
+`CanvasRenderingContext2D` property choose the one with appropriate size; 
+otherwise, it syntheesizes small-caps by using uppercase glyphs. The default 
+value is `normal`.
+
+
+## Syntax
+
+`CanvasRenderingContext2D.fontVariantCaps = "normal" || "small-caps" || "all-small-caps" || "petite-caps" || "all-petite-caps" || "unicase" || "titling-caps"`
+`var fontVariantCaps = CanvasRenderingContext2D.textRendering`
+
+### Values
+
+A `string` representing current `fontVariantCaps` value. The possible values are:
+
+#### Options
+
+`"normal"`
+Deactivates of the use of alternate glyphs.
+
+`"small-caps"`
+Enables display of small capitals (OpenType feature: `smcp`). Small-caps glyphs
+typically use the form of uppercase letters but are reduced to the size of
+lowercase letters.
+
+`"all-small-caps"`
+Enables display of small capitals for both upper and lowercase letters 
+(OpenType features: `c2sc`, `smcp`).
+
+`"petite-caps"`
+Enables display of petite capitals (OpenType feature: `pcap`). petite-caps glyphs
+typically use the form of uppercase letters but are reduced to the size of
+lowercase letters.
```

Comment:
Is this the same as small caps? I don't understand

Reply:
Yes, the values are from different opentype. Visually, they looks the same.